terraform/aws/variables.tf

```diff
@@ -91,3 +91,10 @@ variable "buckets" {
   List of S3 Buckets to create.
   EOT
 }
+
+variable "efs_storage_capacity" {
```

Comment:
EFS has no real 'capacity' and grows as it is needed, no? I don't think this variable has any effect, and I suggest we just hardcode it to 1Mi or something, as we do here (https://github.com/2i2c-org/infrastructure/blob/49e863594b19542f644130875317924a5f0cb42e/helm-charts/basehub/templates/nfs.yaml#L8) (that's also used to mount EFS)

Reply:
sounds good